core/src/main/scala/tech/beshu/ror/accesscontrol/factory/decoders/AuditingSettingsDecoder.scala

```diff
@@ -59,14 +59,26 @@ object AuditingSettingsDecoder extends Logging {
   }
 
   private def decodeAuditSettings = {
+    decodeAuditSettingsWith(auditSinkConfigExtendedDecoder)
```

Comment:
why not `decodeAuditSettingsWith(auditSinkConfigSimpleDecoder).orElse(decodeAuditSettingsWith(auditSinkConfigExtendedDecoder))`?

shouldn't we decode the simple case first?